platform-code/README.md

```diff
@@ -0,0 +1,9 @@
+# Platform Code
+
+All code for the platform development can be found at our [AutSPACEs Repository](https://github.com/alan-turing-institute/AutSPACEs) dedicated to code development only. 
+
+Please add the code you are working on to this repository. 
+
+All other documentation relating to the project needs to be upload to [this repository](https://github.com/alan-turing-institute/AutisticaCitizenScience). 
```

Comment:
All other documentation relating to the project needs to be uploaded to [this repository](https://github.com/alan-turing-institute/AutisticaCitizenScience).